Sources/NFCPassportReader/PassportReader.swift

```diff
@@ -10,11 +10,11 @@ import Foundation
 import OSLog
```

Comment:
Here formatting changes make everything hard to read :S I'm sorry. Let me know if this is still ok to review/merge

Reply:
I'm trying to match the old formatting style via swift-format but it seems paranthesis spaces and other weird character spacing is not part of the official spec. Which tool are you using to format the code?

<img width="862" height="536" alt="CleanShot 2025-12-11 at 09 27 46@2x" src="https://github.com/user-attachments/assets/5a792c3b-3eb8-44eb-b557-eeff2398dd07" />

Reply:
I tried reducing my changes to the minimum but at this point no formatter can replicate the style that was there before. Let me know if you can review the code in this format. Basically at this point besides the code formatting the only other change is the `enforceCA` flag that forces chip authentication to prevent replay attacks. Though I would strongly encourage the use of a formatter.